libs/pipeline.py

```diff
@@ -14,7 +14,7 @@
 
 import pyseir
 from covidactnow.datapublic.common_fields import CommonFields
-from libs.datasets import combined_datasets
+from libs.datasets import combined_datasets, timeseries
```

Comment:
separate lines :)

Reply:
I've now tried asking pycharm to take care of this for me: https://www.jetbrains.com/pycharm/guide/tips/one-import-per-line/

Reply:
👍  I'm planning on running isort across the repo to give our imports a once through clean (I don't generally find it's necessary to force properly sorted imports, but that its helpful to tidy up every once in a while).  at which time could add to the pylint rules as well